example/build_and_run.sh

```diff
@@ -11,7 +11,7 @@ cd ..
 mkdir -p dist
 cd dist
 if [ ! -f "tokenizer.model" ]; then
-    wget https://huggingface.co/decapoda-research/llama-7b-hf/resolve/main/tokenizer.model
+    wget https://huggingface.co/lmsys/vicuna-7b-v1.5/resolve/main/tokenizer.model
```

Comment:
wget cannot download decapoda-research/llama-7b-hf without logging in. But vicuna-7b-v1.5 does not have this restriction.